docs/user-guide/kubernetes.md

```diff
@@ -21,6 +21,9 @@ If your cluster is configured with RBAC, you will need to authorize Træfik to u
 
 RoleBindings per namespace enable to restrict granted permissions to the very namespaces only that Træfik is watching over, thereby following the least-privileges principle. This is the preferred approach if Træfik is not supposed to watch all namespaces, and the set of namespaces does not change dynamically. Otherwise, a single ClusterRoleBinding must be employed.
 
+!!! note
+    RoleBindings per namespace will be available in Træfik 1.5+ please use ClusterRoleBinding for any older version.
```

Comment:
Can we rewrite this as following:

_RoleBindings per namespace are available in Træfik 1.5 and later. Please use ClusterRoleBindings for older versions._

(Present tense; replace plus by verbatim text; second sentence; plural for ClusterRoleBinding too; older versions.)

Reply:
Sure sounds good. From a documentation perspective in the 1.5 branch present tens makes totally sense. The rest is fine with me. I'll try to make the changes :)